lib/BUILD.bazel

```diff
@@ -0,0 +1,38 @@
+load("@npm_bazel_typescript//:defs.bzl", "ts_library")
+package(default_visibility = ["//:__pkg__"])
+
+exports_files([
+    "zone.ts",
```

Comment:
exposing sources outside the package is a smell, usually any rules using them should be declared here too. At least add a comment explaining why another package needs to be able to use these sources?
(the `.ts` files in particular seem wrong here, they should be compiled by a rule in this package)

Reply:
This for test, I have removed it and will re-design when we figure out how to make `karma` for zone.js work.